packages/components/src/form-toggle/styles/form-toggle-styles.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import styled from '@emotion/styled';
+import { css } from '@emotion/core';
+
+/**
+ * Internal dependencies
+ */
+import { color, reduceMotion, rtl, zIndex } from '../../utils';
+import {
+	toggleBorderWidth,
+	toggleWidth,
+	toggleHeight,
+	appearTransform,
+} from './utils';
+
+export const StyledToggleTrack = styled.span`
+	content: '';
+	display: inline-block;
+	box-sizing: border-box;
+	vertical-align: top;
+	background-color: ${ color( 'white' ) };
+	border: ${ toggleBorderWidth }px solid ${ color( 'darkGray.primary' ) };
+	width: ${ toggleWidth }px;
+	height: ${ toggleHeight }px;
+	border-radius: ${ toggleHeight / 2 }px;
+	transition: 0.2s background ease;
+	${ reduceMotion( 'transition' ) }
+`;
+
+export const StyledToggleThumb = styled.span`
+	display: block;
+	position: absolute;
+	box-sizing: border-box;
+	top: ${ toggleBorderWidth * 3 }px;
+	${ rtl( { left: `${ toggleBorderWidth * 3 }px` } ) };
+	width: ${ toggleHeight - toggleBorderWidth * 6 }px;
+	height: ${ toggleHeight - toggleBorderWidth * 6 }px;
+	border-radius: 50%;
+	transition: 0.1s transform ease;
+    ${ reduceMotion( 'transition' ) }
```

Comment:
Indentation looks funky here. I wonder if something is up with the linter (or prettier?) not being able to catch/format this 🤔